src/lang/english.txt

```diff
@@ -988,6 +988,7 @@ STR_GAME_OPTIONS_CURRENCY_TRY                                   :Turkish Lira
 STR_GAME_OPTIONS_CURRENCY_SKK                                   :Slovak Koruna
 STR_GAME_OPTIONS_CURRENCY_BRL                                   :Brazilian Real
 STR_GAME_OPTIONS_CURRENCY_EEK                                   :Estonian Krooni
+STR_GAME_OPTIONS_CURRENCY_LVL                                   :Latvian Lats
```

Comment:
Order of the strings should also match :)

Reply:
fixed string order 🤦🏼‍♂️